src/modelfree/tsne/visualize.py

```diff
@@ -0,0 +1,182 @@
+import glob
+import logging
+import os
+import os.path as osp
+import tempfile
+
+import matplotlib
+import matplotlib.pyplot as plt
+import numpy as np
+import pandas as pd
+import sacred
+from sacred.observers import FileStorageObserver
+
+from modelfree.common import utils
+from modelfree.visualize.styles import STYLES
+
+visualize_ex = sacred.Experiment('tsne_visualize')
+logger = logging.getLogger('modelfree.tsne.visualize')
+
+
+@visualize_ex.config
+def main_config():
+    model_glob = None
```

Comment:
Is this the root directory where your policies/models lived? If so, I have a preference for it being named something like "models_root" rather than model_glob, since the latter isn't really clear as to what it is

Reply:
It's a glob, you can specify e.g. `models/SumoAnts*` and it will just rerun it for `SumoAnts`. Added comment to clarify.